src/main/java/com/google/sps/servlets/utils/YouTubeCommentRetriever.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.servlets.utils;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.*;
```

Comment:
I'm okay with it in this case since your IDE support currently isn't the best for managing imports, but generally you should avoid wildcard imports.

One primary benefit is that readers of your code will know where each class/method you consume comes from.

If for example you have multiple wildcard imports, it will be very difficult to determine the package associated w/ the different classes/methods in use.

Reply:
Got it, the reason I used the wildcard import was because of the amount of things that would otherwise have to be imported.